tests/search/test-class-search.php

```diff
@@ -488,4 +488,57 @@ public function test__vip_search_setup_healthchecks_disabled_in_non_production_e
 		// Should not have instantiated and registered the init action to setup the health check
 		$this->assertEquals( false, $es->healthcheck->is_enabled() );
 	}
+
+	/**
+	 * Test that checks both single and multi-host retries
+	 */
+	public function test__vip_search_filter__ep_pre_request_host() {
+		$es = new \Automattic\VIP\Search\Search();
+		$es->init();
+
+		// If VIP_ELASTICSEARCH_ENDPOINTS is not defined, just hand the last host back
+		$this->assertEquals( 'test', $es->filter__ep_pre_request_host( 'test', 0, '', array() ) );
```

Comment:
I realize that we haven't been following this on the existing tests in this file, but as a general rule, whenever you have multiple assertions in the same test, it's helpful to add the `$message` param to indicate which assertion failed and why.

```suggestion
		$this->assertEquals( 'test', $es->filter__ep_pre_request_host( 'test', 0, '', array() ), 'pre_request_host filter did not return passed in host without constant defined.' );
```

This way, when a test fails, we get more obvious details around which assertion was the problem.